lib/github_changelog_generator/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module GitHubChangelogGenerator
-  VERSION = "1.16.4"
+  VERSION = "1.17.0"
```

Comment:
```suggestion
  VERSION = "1.16.4"
```

The release process takes care of updating the version number. Don't change this in non-release PRs.

Reply:
No biggie, I can resolve this.